projects/editions/pkg/configure-dfm/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:edge@sha256:99588bc8883c955c157d18fc3eaa4a3c1400c223e6c7cabca5f600a3e9f8d5cd
+ENTRYPOINT []
```

Comment:
we should use the standard way we do this elsewhere, from the linuxkit/alpine base